addons/account/models/account_cash_rounding.py

```diff
@@ -29,8 +29,8 @@ class AccountCashRounding(models.Model):
     @api.constrains('rounding')
     def validate_rounding(self):
         for record in self:
-            if record.rounding < 0:
-                raise ValidationError(_("Please set a positive rounding value."))
+            if record.rounding <= 0:
+                raise ValidationError(_("Please set a strictly positive rounding value."))
```

Comment:
Shouldn't you update the .pot as well, then?

Reply:
Damn, I was expecting you'd miss that :D